addons/mail/static/src/new/thread/link_preview/link_preview_list.js

```diff
@@ -0,0 +1,50 @@
+/** @odoo-module */
+
+import { useMessaging } from "../../messaging_hook";
+import { Component } from "@odoo/owl";
+import { LinkPreviewCard } from "./link_preview_card";
+import { LinkPreviewImage } from "./link_preview_image";
+import { LinkPreviewVideo } from "./link_preview_video";
+
+export class LinkPreviewList extends Component {
```

Comment:
Might be worth adding prefix `Message`, so that it makes sense this component is intended to show the link previews of a message, as it's the main `prop` of this component.

Reply:
Message is not the main props anymore.
This component should not be limited to message, but work with what ever link preview you are passing as props.

Reply:
Ok for me, as long as there's coherence between name of component and its props